db/migrations/versions/042_remove_old_reporting_round_d.py

```diff
@@ -0,0 +1,35 @@
+"""remove old reporting round data
```

Comment:
Maybe slightly misnamed as the migration doesn't remove data, just makes the columns nullable